test/api/buffer/receivers/tasks/initial-etcd-kvs.txt

```diff
@@ -0,0 +1,27 @@
+<<<<<
+/config/receiver/%%TEST_KBC_PROJECT_ID_8DIG%%/my-receiver
+-----
+{
+  "projectId": %%TEST_KBC_PROJECT_ID%%,
+  "receiverId": "my-receiver",
+  "name": "My Receiver",
+  "secret": "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+}
+>>>>>
+
+<<<<<
+task/%%TEST_KBC_PROJECT_ID_8DIG%%/my-receiver/receiver.create/2023-01-01T08:02:06.000Z_abcdef
```

Comment:
@JakubMatejka tak asi k tomu nemam nic viac, len ze podla schemy je to `/task` a nie `task`.

Reply:
po oprave by to uz malo ist